client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx

```diff
@@ -88,13 +89,32 @@ const SiteCreationStep: Step = function SiteCreationStep( { navigation, flow } )
 		};
 	}
 
+	const [ freeFlowDomainCartItemLoaded, setFreeFlowDomainCartItemLoaded ] = useState( false );
```

Comment:
**Note**: Wondering if folks have thoughts about better ways to hydrate state with a dynamically generated domain name